src/Uno.UI/UI/Xaml/DragDrop/DropUITarget.cs

```diff
@@ -131,7 +131,7 @@ public IAsyncOperation<DataPackageOperation> DropAsync(CoreDragInfo dragInfo)
 				dragInfo.Position,
 				Window.Current.RootElement.XamlRoot, //TODO: Choose proper XamlRoot https://github.com/unoplatform/uno/issues/8978
 				getTestability: GetDropHitTestability,
-				isStale: elt => elt.IsDragOver(dragInfo.SourceId));
+				isStale: new (elt => elt.IsDragOver(dragInfo.SourceId), "IsDragOver"));
```

Comment:
I usually prefer explicit type when type isn't clear just by looking.

```suggestion
				isStale: new StalePredicate(elt => elt.IsDragOver(dragInfo.SourceId), "IsDragOver"));
```